### ISSUE TITLE
Track popular requests

### ISSUE THREAD

**Issue by Popolechien** [openzim/zimit-frontend#47]
This was a soft requirement from the sponsor of the zimit 2.0 update: but we should be able to identify recurring requests over a given period of time.

**Comment by rgaudin** [comment]
Difficulty here is to define what is a recurring request. Is domain name enough? We see many requests for subreddits in youzim.it for instance

**Comment by Popolechien** [comment]
Top-level URL should be enough (so https://www.reddit.com/ rather than reddit.com/r/fubar in your example)

**Comment by benoit74** [comment]
Is it ok if we report only `www.reddit.com` in your example (instead of `https://www.reddit.com`?). It would be very easy to report this manually once in a while with a small SQL query based on recipe names.

**Comment by benoit74** [comment]
And do you need to have this in a UI, or is it something which could be done manually by a developer.

I imagine we could have an already prepared SQL query which would build the list of domain names over a given period, and it would "consume" only few minutes for a dev to run it upon an inquiry from someone.

**Comment by kelson42** [comment]
I would propose to implement an export of task in CSV and then this kind of sorting/filtering can be done in a third party software.

**Comment by benoit74** [comment]
Should we move this issue to zimfarm then? Or does it needs to be available in zimit UI? (I don't get who will use this feature)

**Comment by Popolechien** [comment]
@benoit74 Yes probably a good idea to move it to zimfarm, and yes for reddit.com instead of whatever longer and equally informative alternative.
